internal/services/automation/automation_runbook_resource.go

```diff
@@ -22,6 +23,49 @@ import (
 	"github.com/hashicorp/terraform-provider-azurerm/utils"
 )
 
+func contentLinkSchema(isDraft bool) *pluginsdk.Schema {
+	ins := &pluginsdk.Schema{
+		Type:     pluginsdk.TypeList,
+		Optional: true,
+		MaxItems: 1,
+		Elem: &pluginsdk.Resource{
+			Schema: map[string]*pluginsdk.Schema{
+				"uri": {
+					Type:     pluginsdk.TypeString,
+					Required: true,
+				},
```

Comment:
can we validate this?

Reply:
the `uri` can be empty and there is no validation before, so I removed the validation.

Reply:
so this it is either empty or a url? can we validate that then? validation.Any(url + empbty?) ?